Bio/Phylo/TreeConstruction.py

```diff
@@ -372,22 +372,30 @@ class DistanceCalculator(object):
 
     Examples
     --------
-    >>> from Bio.Phylo.TreeConstruction import DistanceCalculator
-    >>> from Bio import AlignIO
-    >>> aln = AlignIO.read(open('TreeConstruction/msa.phy'), 'phylip')
-    >>> print(aln)
-    SingleLetterAlphabet() alignment with 5 rows and 13 columns
-    AACGTGGCCACAT Alpha
-    AAGGTCGCCACAC Beta
-    CAGTTCGCCACAA Gamma
-    GAGATTTCCGCCT Delta
-    GAGATCTCCGCCC Epsilon
+    ::
```

Comment:
That's double colon alone is a bit ugly, how about ``Loading a small PHYLIP alignment from which to compute distances::``